backend/src/StdLibDarkInternal/Helpers/ProgramTypesToDarkTypes.fs

```diff
@@ -715,8 +715,12 @@ module Expr =
 
 
       // control flow
-      | PT.EIf(id, cond, ifTrue, ifFalse) ->
-        "EIf", [ DInt(int64 id); toDT cond; toDT ifTrue; toDT ifFalse ]
+      | PT.EIf(id, cond, thenExpr, elseExpr) ->
+        let elseExpr =
+          match elseExpr with
+          | Some elExp -> elExp
+          | None -> PT.EUnit id
```

Comment:
For Dark types, we use full proper options. See for example `EnumField.label`

Reply:
hm, it is not clear for me what does mean "full proper options" here. 
if you refer to [EnumField toDT](https://github.com/darklang/dark/blob/main/backend/src/StdLibDarkInternal/Helpers/ProgramTypesToDarkTypes.fs#L972) then it seems like we need simple unwrap the optional - that is we exactly to do here. Please clarify.

Reply:
I think Paul is suggesting something like (maybe exactly):
```fsharp
let elseExpr =  elseExpr |> Option.map Expr.toDT |> Dval.option
```

Reply:
aa ok, got it, thanks @StachuDotNet